tests/phpunit/class-wc-stripe-order-handler-test.php

```
@@ -105,6 +105,132 @@ function () use ( &$fired ) {
+			]
+		);
+		$this->assertNotEmpty(
+			array_filter(
+				$notes,
+				function ( $note ) {
+					return false !== strpos( $note->content, 'instead of being auto-cancelled' );
+				}
+			),
+			'A note should explain why the cancellation was blocked.'
+		);
+	}
```

PHPMD (2.15.0)

[error] The method test_settles_order_with_paid_intent_instead_of_cancelling is not named in camelCase. (undefined)

(CamelCaseMethodName)